code/modules/organs/external/_external_icons.dm

```diff
@@ -1,7 +1,10 @@
 var/list/limb_icon_cache = list()
 
-/obj/item/organ/external/set_dir()
-	return
+/obj/item/organ/external/set_dir(var/direction, var/forced)
```

Comment:
This allows external organs to forcefully set a direction when desired, while preventing normal attempts